asteroid.js

```diff
@@ -3,22 +3,33 @@
 // http://patreon.com/codingrainbow
 // Code for: https://youtu.be/hacZU523FyM
 
-function Asteroid(pos, r) {
-  if (pos === undefined) {
-    pos = createVector(random(width), random(height))
+function Asteroid(pos, r, size) {
+  if (!pos) {
+    pos = createVector(random(width), random(height));
   }
 
-  r = r !== undefined ? r * 0.5 : random(15, 30);
+  r = r ? r * 0.5 : random(40, 60);
```

Comment:
this will not work if the radius is 0 but I guess that is fine since having a 0 as a radius is bad anyway.